modelzoo/resnet50.py

```diff
@@ -0,0 +1,192 @@
+import lasagne
```

Comment:
done
>>The pickle file doesn't need to be downloaded for the notebook at all, does it?
yes, you are right; pkl file is created by this notebook, notebook demonstrates how resnet works and how to copy weights from Caffe with respect to different implementation of some layers
then pkl file can be used for transfer weight to model created by modelzoo/resnet50.py